zebra-scan/src/tests.rs

```diff
@@ -550,3 +509,47 @@ fn random_compact_tx(mut rng: impl RngCore) -> CompactTx {
     ctx.outputs.push(cout);
     ctx
 }
+
+/// Returns transactions belonging to any of the given [`ScanningKey`]s.
+///
+/// TODO:
+/// - Remove the `sapling_tree_size` parameter or turn it into an `Option` once we have access to
+/// Zebra's state, and we can retrieve the tree size ourselves.
+/// - Add prior block metadata once we have access to Zebra's state.
+fn scan_block<K: ScanningKey>(
+    network: Network,
+    block: Arc<Block>,
+    sapling_tree_size: u32,
+    scanning_keys: &[&K],
+) -> Result<ScannedBlock<K::Nf>, ScanError> {
+    let network: zcash_primitives::consensus::Network = match network {
+        Network::Mainnet => zcash_primitives::consensus::Network::MainNetwork,
+        Network::Testnet => zcash_primitives::consensus::Network::TestNetwork,
+    };
+
+    // TODO: Implement a check that returns early when the block height is below the Sapling
+    // activation height.
+
+    let chain_metadata = ChainMetadata {
+        sapling_commitment_tree_size: sapling_tree_size,
+        orchard_commitment_tree_size: 0,
```

Comment:
Optional: 

Add some comment on why we have this value set to 0.

```suggestion
        // Orchard is not supported at the moment so the tree size can be 0.
        orchard_commitment_tree_size: 0,
```